### ISSUE TITLE
Refreshrate resets to default after `power off` & `power on`

### ISSUE THREAD

**Issue by litoj** [swaywm/sway#8297]
### Please fill out the following:
- **Sway Version:** 1.9
- **Debug Log:**
  - [sway.log](https://github.com/user-attachments/files/16623116/sway.log)
- **Configuration File:**
```swayconfig
output eDP-1 mode 2560x1600@48Hz
bindsym t output eDP-1 power toggle
bindsym q exit
```
- **Description:**
  - when sway is started, the output is applied correctly, but when the config changes or the display is turned off and on (swayidle `swaymsg power off/on`) the refreshrate setting doesn't get applied again (it uses the default), but other settings like resolution and scaling do.
  - I think sway needs to request the refreshrate always when turning the display on, not just on the first connection.

**Comment by nekopsykose** [comment]
does this reproduce on master? i remember having a similarish bug that i remember maybe got fixed but lost track of it..

**Comment by litoj** [comment]
Haven't checked yet, will do and report back.

**Comment by etrnal70** [comment]
for me just reloading sway config also resets the refresh rate

**Comment by sahinf** [comment]
> the refreshrate setting doesn't get applied again (it uses the default),

Doesn't your log say it does get applied again? ctrl+f for `Hz` and they're all 48 except the detected modes section `00:00:01.398 [INFO] [wlr] [backend/drm/drm.c:786] connector eDP-1: Modesetting with 2560x1600 @ 48.000 Hz`

**Comment by litoj** [comment]
Yes, that is the very core of the issue - when the screen is turned back on, it prints info about the last uset setting that it thinks is also the current. And continues to report this as the current setting despite clearly running on defaults. I saw this with mpv (said system uses 48Hz) and <https://fpstest.org/refresh-rate-test/> (reported 120Hz when actually testing the screen refreshrate)
